src/views/tutorial-library/index.tsx

```diff
@@ -26,18 +34,36 @@ const queryHook: UseSearchBoxProps['queryHook'] = (query, search) => {
 export default function TutorialLibraryView() {
 	const { query: searchQuery, refine } = useSearchBox({ queryHook })
 	const [query, setQuery] = useState<string>(searchQuery)
+	const [showMobileFilters, setShowMobileFilters] = useState(false)
+	const filtersState = useFiltersState()
 
 	return (
 		<div>
 			<h1>Tutorial Library</h1>
-			<FilterInput
-				placeholder="Filter results"
-				value={query}
-				onChange={(value) => {
-					setQuery(value)
-					refine(value)
-				}}
-			/>
+			<div className={s.inputFilterSection}>
+				<FilterInput
+					className={s.input}
+					placeholder="Filter results"
+					value={query}
+					onChange={(value) => {
+						setQuery(value)
+						refine(value)
+					}}
+				/>
+				<MobileFiltersButton onClick={() => setShowMobileFilters(true)} />
+				<Dialog
+					isOpen={showMobileFilters}
+					label="Tutorial filters"
+					onDismiss={() => setShowMobileFilters(false)}
+					variant="bottom"
+				>
+					<div className={s.mobileFiltersControls}>
+						<Button text="Done" onClick={() => setShowMobileFilters(false)} />
+						<ClearFilters />
+					</div>
+					<TutorialLibraryFilters {...filtersState} />
```

Comment:
Passing the filters state down like this fixes the issue where search state is cleared when the dialog is closed, as it persists at the view component level.